.storybook/addons/props/About.js

```diff
@@ -6,19 +6,37 @@ import getImportPath from './getImportPath';
 import getSourcePath from './getSourcePath';
 import PathBar from './PathBar';
 import PropTable from './PropTable';
+import LogTable from './LogTable';
 
 const Wrapper = styled.div({
   padding: 16,
   fontSize: 14,
 });
 
+const Header = styled.h2({
+  marginBottom: 16,
+  marginTop: 24,
+});
+
+const SubHeader = styled.h4(({ theme }) => ({
+  float: 'right',
```

Comment:
need to clear this float?

Reply:
Doesn't need clearing since the container is larger than the floated content.

Reply:
siiiick just checking